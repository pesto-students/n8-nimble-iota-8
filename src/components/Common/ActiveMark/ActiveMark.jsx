import React from "react";
import styles from "src/components/Common/ActiveMark/ActiveMark.module.less";

const ActiveMark = () => <div className={styles.activeMark}></div>;

export default ActiveMark;
