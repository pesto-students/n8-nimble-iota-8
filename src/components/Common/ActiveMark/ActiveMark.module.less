.activeMark {
    width: 0.75em;
    height: 0.75em;
    border-radius: 50em;
    background-color: rgba(#2ffa43, 0.54);
    display: inline-block;
}
