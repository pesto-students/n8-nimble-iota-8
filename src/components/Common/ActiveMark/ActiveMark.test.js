import React from "react";
import renderer from "react-test-renderer";
import ActiveMark from "src/components/Common/ActiveMark/ActiveMark";
it("ActiveMark renders correctly", () => {
    const tree = renderer.create(<ActiveMark />).toJSON();
    expect(tree).toMatchSnapshot();
});
