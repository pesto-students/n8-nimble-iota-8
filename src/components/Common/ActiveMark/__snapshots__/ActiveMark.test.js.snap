// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ActiveMark renders correctly 1`] = `<div />`;
